drivers/health_comprehensive_assessment/app/models/health_comprehensive_assessment/document_exports/health_ca_pdf_export.rb

```diff
@@ -98,15 +98,15 @@ def perform
           },
         )
 
-        pdf = CombinePDF.new
-        pdf << CombinePDF.parse(PdfGenerator.new.render_pdf(first_page_html, options: first_page_options), allow_optional_content: true)
-        pdf << CombinePDF.parse(PdfGenerator.new.render_pdf(html, options: body_options), allow_optional_content: true)
+        pdf = []
+        pdf << PdfGenerator.merge_inline_pdfs(PdfGenerator.new.render_pdf(first_page_html, options: first_page_options))
+        pdf << PdfGenerator.merge_inline_pdfs(PdfGenerator.new.render_pdf(html, options: body_options))
```

Comment:
this is fine although  maybe for consistency we should drop a pass-through class method on PdfGenerator for render_pdf so we aren't mixing instance/class methods
```ruby
class PdfGenerator
  def render_pdf(...)
    new.render_pdf(...)
  end
end
```

```suggestion
        pdf = []
        
        pdf << PdfGenerator.merge_inline_pdfs(PdfGenerator.render_pdf(first_page_html, options: first_page_options))
        pdf << PdfGenerator.merge_inline_pdfs(PdfGenerator.render_pdf(html, options: body_options))
```

also, could dry this up a bit. Maybe not worth it though
```ruby
pdf = [
  [first_page_html, first_page_options],
  [html, body_options]
].map do |html, options |
   PdfGenerator.merge_inline_pdfs(PdfGenerator.render_pdf(html, options: options))
end
```